prow/cmd/config-bootstrapper/main.go

```diff
@@ -93,11 +100,23 @@ func main() {
 		logrus.WithError(err).Fatal("Error starting plugin configuration agent.")
 	}
 
-	kubeClient, err := o.kubernetes.Client(configAgent.Config().ProwJobNamespace, o.dryRun)
+	credentials, err := clientcmd.NewDefaultClientConfigLoadingRules().Load()
```

Comment:
Do we want to make this work for GKE clusters?

https://github.com/kubernetes/test-infra/blob/e18e702637604781654fe3b0901d5412ccccd028/prow/cmd/build/main.go#L48

Reply:
Sure